lib/component/catalog_table.es6.js

```diff
@@ -0,0 +1,133 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+function catalogTableController($scope, api) {
```

Comment:
But in the bottom of the file,  I am using the array syntax to add this controller to component
`controller: ['$scope', 'api', catalogTableController]`
Is that what you suggests?